frontend/src/features/permits/pages/TermOversize/components/pay/PermitPayFeeSummary.tsx

```diff
@@ -14,6 +16,14 @@ export const PermitPayFeeSummary = ({
   permitType?: PermitType;
   onPay: () => void;
 }) => {
+  const [inputAction, setInputAction] = useState(false);
+  const debouncedInputAction = useDebounce(inputAction, DEBOUNCE_TIMEOUT);
+
+  useEffect(() => {
+    if (debouncedInputAction) { onPay(); }
+
```

Comment:
Probably need to fix the formatting here...

Reply:
updated